dev-lang/opensmalltalk_vm/opensmalltalk_vm-202312181441.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Cross-platform virtual machine for Squeak, Pharo, and Cuis"
+DESCRIPTION="opensmalltalk-vm (a.k.a. the Cog VM) is the virtual machine beneath the Cuis, \
+Pharo and Squeak Smalltalk dialects."
+HOMEPAGE="https://opensmalltalk.org/"
+COPYRIGHT="2002-2025 OpenSmalltalk developers"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/OpenSmalltalk/opensmalltalk-vm/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="3d5dd4dc8f6ffd5af6535479a84081442319b738d6c580dae043b7b0069e73cd"
+SOURCE_DIR="opensmalltalk-vm-$portVersion"
+PATCHES="opensmalltalk_vm-$portVersion.patchset"
+
+ARCHITECTURES="x86_64"
+
+PROVIDES="
+	opensmalltalk_vm = $portVersion
+	cmd:squeak
+	"
+REQUIRES="
+	haiku
+	lib:libX11
+	lib:libssl
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	xlibe_devel
+	devel:libssl
+	perl
```

Comment:
Not required:
```suggestion
-	perl
```

Reply:
I'm guessing `cmd:perl` is required on runtime though?

Reply:
No. I'll remove this.

Reply:
You don't need perl to run the vm, but I believe it's required to compile it.